src/bentoml/models.py

```diff
@@ -115,6 +121,46 @@ def import_model(
     Returns:
         Model: the imported model
     """
+    parsedURL = urlparse(path)
+    if (
+        parsedURL.scheme == "http" or parsedURL.scheme == "https"
+    ) and parsedURL.netloc == "huggingface.co":
+        model_id = parsedURL.path.strip("/")
```

Comment:
I assume we are making a specific branch for huggingface here? Should we use the `bentoml.transformers.import_model` instead? 

Ofc once `bentoml.transformers.import_model` is ready.